pyproject.toml

```diff
@@ -69,6 +69,21 @@ exclude = ["binder", "docker", "docs", "tests", "examples"]
 [tool.setuptools_scm]
 fallback_version = "0+untagged"
 
+[tool.ruff]
+line-length = 90
+
+[tool.ruff.lint]
+# Add the following rule sets
+select = ["E", "W", "F", "B", "UP", "SIM", "RUF022"]
+# But ignore these inconvenient rules
+ignore = ["F403", "E226", "E731", "E275", "F405", "E722", "E741", "W605"]
```

Comment:
Would it be worth adding a readme somewhere (could even just be a link to relevant documentation in a comment) showing how to configure their own local Ruff according to this setup?